### PR TITLE
Cleanup A records: add IAM permissions for EC2, Route53, auto scaling to run cron job

### PR DESCRIPTION
**Pull request by robinkwilson**
Added permissions to "app", "app,stream" boxes for running cleanup_records cron job.

IAM Permissions added:
- EC2 describe-instances
- Route53 describe records
- Route53 changeset to delete records
- Auto scaling describe auto scaling groups

### CODE REVIEW

**Comment by robinkwilson** [comment]
## QA Test Plan

### TLDR PR context
Issue: When a EC2 instance is deleted, it leaves behind an A record on the Hosted Zone to a subdomain with an IP address that no longer maps to any of your instances, this poses a security risk. 

To verify the current problem, both: open Route53 Hosted Zone and select the internal domain and open your EC2 instances. Click on an EC2 box (ex hostname `optimistic-invoker`) > Click "instance status" > Select "Terminate". Check Route53 Hosted Zones and search for `optimistic-invoker` in the records list. Despite being terminated, that record still maps to the old IP address.

This template change deletes the A records added to a domain (if the domain has a Hosted Zone on AWS). This runs every hour -- _I will provide templates for the hour and one that runs every 5 minutes for testing purposes via email_

**_I will combine all of these template changes into one for a thorough hubs test._**

At a minimum for testing just this PR change:
- Create a new stack or update an existing one with new template
- Terminate instances, update stack instance sizes, or change the number of instances for a multi-server setup
- Check if the domain is hosted on another host (Recipe 3) still works (it should since the internal domain should have a hosted zone on aws and that's what we mess with)

## Plan

I will provide template links via email:
- Personal: every 1 hour cron job (prod) and every 5 minutes cron job (testing)
- Enterprise multi-server: every 1 hour cron job (prod) and every 5 minutes cron job (testing)

### Update existing stack with a new template (users with existing stacks will follow this workflow)
#### For Personal and Enterprise multi-server
1. AWS Cloudformation
2. Click on your stack
3. Click "Update"
4. Select "Replace current template"
5. Paste in new template URL
6. Click "Next"
7. No need to change any stack parameters so scroll through and finish the update (should have recipe 3 and 2 to use)
8. Deploy then follow "Verify below cases work for each deployment"

### New stacks, create a new stack with template
#### For Personal and Enterprise multi-server
1. AWS Cloudformation
2. Click "Create stack"
3. Paste in new template URL
4. Fill in params for Recipe 1, 2, 3
5. Deploy then follow "Verify below cases work for each deployment"

### Verify below cases work for each deployment
This will involve a lot more cloudformation updates / EC2 updates than usual Hubs software stuff.
1. Deploy the stacks and ensure you can still do "the usual Hubs stuff" (see below "The usual Hubs stuff")
2. Test that the 




### "The usual Hubs stuff"
- Can login
- Can join a room
- Can hear people in a room, can change avatar, can move around, can add assets to a room
- Can make a scene in spoke

 
1. Run stack
2. Create room
3. Join room with 2 or more people
   1. Use query params `?force_tcp=true`, `?force_turn=true`, as well as `?force_turn=true&force_tcp=true`
   2. Use no query params
   3. Try blocking port 80 tcp and udp (I used windows firewall to block 80 for my tests)
4. Ensure every person can enter the room, hear everyone, move around, bring in images and assets ("the usual hubs stuff")
5. Ensure you can login
6. Setup ssh and ensure you can ssh into the box (make sure we didn't mess up port 22)
   1. In the admin panel add 2FA in Server Access
   2. SSH into box
7. Success, awesome! Done verified :D 

THANK YOU!! You're awesome! Happy to answer any questions around this :)
